frontend/src/views/StartView.vue

```diff
@@ -0,0 +1,228 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import { Breadcrumb, Button, Card, RadioButton } from '@/lib/primevue';
+import { RouteNames } from '@/utils/constants';
+import type { Ref } from 'vue';
```

Comment:
Our spacing for imports is a bit specific. See FileUpload.vue for a small example.

```
3rd party imports

application imports

3rd party types
application types
```

Always alphabetized. The ordering here is correct, just missing the new lines.

Reply:
Thank you!